questions/Changing HTML Attribute dynamically

```diff
@@ -0,0 +1,20 @@
+### Can we modify an attribute's value dynamically for an HTML tag? 
+
+#### Answer
+
+Yes. It is possible by using javascript using 'getElementById' method.
```

Comment:
there are many ways to resolve to a DOM node object in JS and thus should be mentioned that this is but one of many ways.

Reply:
Another alternative to this would be to explain that you can use the Element class attributes in JS to modify them dynamically and avoid mentioning how to find it in the DOM.